app/controllers/submissions_controller.rb

```diff
@@ -21,7 +21,7 @@ def edit_remix
 
   def update
     find_submission
-    if @submission.update_attributes(submission_params)
+    if @submission.update_attributes(submission_params) && @submission.remix_id != nil
```

Comment:
Line is too long. [86/80]<br>Prefer `!expression.nil?` over `expression != nil`.